### PR TITLE
Feature#12/create the header layout

### PR DESCRIPTION
**Pull request by nallely64**
# Description
 - What?: I've added the responsive header  layout for the views 
 - Why?: These changes are helpfull for the user to find products and to navigate in the menu.

  ## Screenshots 
Captura de pantalla de 2022-05-11 14-37-27

Captura de pantalla de 2022-05-11 14-40-32

### CODE REVIEW

File: app/assets/stylesheets/utils/_variables.scss [hunk 1]
```diff
@@ -20,3 +21,6 @@ $text-black-border: $black 1px 0px 0px, $black 0.540302px 0.841471px 0px,
   $black -0.416147px 0.909297px 0px, $black -0.989992px 0.14112px 0px,
   $black -0.653644px -0.756802px 0px, $black 0.283662px -0.958924px 0px,
   $black 0.96017px -0.279415px 0px;
+
+  // Font Family
+  $font: "Roboto", sans-serif;
```
Comment (germanruzca) [inline app/assets/stylesheets/utils/_variables.scss:26]: Add a newline after this

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -15,8 +15,14 @@
...
       <p class="alert"><%= alert %></p>
+      <%= render "layouts/header" %>
       <%= yield %>
       <%=render 'layouts/footer' %>
     </main>  
+
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.3/dist/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+
```
Comment (germanruzca) [inline app/views/layouts/application.html.erb:26]: You don't need this, bootstrap is already added with **_yarn_**.

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+      <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+        <div class="navbar-nav">
+            <a class="nav-item nav-link active" href="#">SERVICIOS</a>
+            <a class="nav-item nav-link active" href="#">PRODUCTOS</a>
+        </div>
+      </div>
+
+      <input type="text" class="form-control icon_placeholder d-inline-block align-top " placeholder=" Busca productos, servicios">
+      <a class="navbar-icon-cart d-inline-block align-top"></a>
```
Comment (germanruzca) [inline app/views/layouts/_header.html.erb:19]: Looks great, but could add the [fontawesome icon cart](https://fontawesome.com/icons/cart-shopping?s=solid)

**Review by germanruzca — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/utils/_variables.scss (hunk 1 above)
Comment (nallely64) [reply app/assets/stylesheets/utils/_variables.scss]: Done!

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (nallely64) [reply app/views/layouts/application.html.erb]: Done!

File: app/views/layouts/_header.html.erb (hunk 3 above)
Comment (nallely64) [reply app/views/layouts/_header.html.erb]: Done! thanks!

**Review by germanruzca — APPROVED** [review]
LGTM, great work 🚀

**Review by juperameza — DISMISSED** [review]
LGTM!!

**Review by AlejandroCuriel — APPROVED** [review]
LGTM :rock: :love_letter:

File: app/assets/stylesheets/utils/_variables.scss [hunk 4]
```diff
@@ -23,3 +24,6 @@ $text-black-border: $black 1px 0px 0px, $black 0.540302px 0.841471px 0px,
   $black -0.416147px 0.909297px 0px, $black -0.989992px 0.14112px 0px,
   $black -0.653644px -0.756802px 0px, $black 0.283662px -0.958924px 0px,
   $black 0.96017px -0.279415px 0px;
+
+// Font Family
+$font: "Roboto", sans-serif;
```
Comment (Javi1499) [inline app/assets/stylesheets/utils/_variables.scss:29]: We will be using different fonts in this project and this name could lead to confusion, you can change "font" to primaryFont, titleFont, etc.

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -17,8 +17,9 @@
       <p class="alert"><%= alert %></p>
     </main>
     <div class="main-container">
+      <%= render "layouts/header" %>
```
Comment (Javi1499) [inline app/views/layouts/application.html.erb:20]: move your partial out of the main container
```suggestion
    <%= render "layouts/header" %>
    <div class="main-container">
```

**Review by Javi1499 — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/utils/_variables.scss (hunk 4 above)
Comment (nallely64) [reply app/assets/stylesheets/utils/_variables.scss]: done! :)

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (nallely64) [reply app/views/layouts/application.html.erb]: done! thanks!

**Review by Javi1499 — APPROVED** [review]
LGTM 👍🏾

**Review by Javi1499 — APPROVED** [review]

**Review by germanruzca — APPROVED** [review]
LGTM

**Pull request merged**